searx/plugins/__init__.py

```diff
@@ -22,7 +22,7 @@
 
 logger = logger.getChild('plugins')
 
-from searx.plugins import (doai_rewrite,
+from searx.plugins import (oa_doi_rewrite,
```

Comment:
What is the benefit of this rename? What does `oa` mean?

Reply:
OA stands for Open Access, _i.e._ scientific publications that are _at least_ available on the web without a paywall (webpage handling payment of a fee to access another webpage).

doia.net is one DOI resolver service, oadoi.org, doi.org are others. The plugin should have a name that is generic enough.

The plugin could be renamed to doi_resolver_rewrite too.